boards/DCM/Src/App/App_DcmWorld.c

```diff
@@ -27,6 +28,7 @@ struct DcmWorld *App_DcmWorld_Create(
     struct Buzzer *const            buzzer,
     struct ErrorTable *const        error_table,
     struct Clock *const             clock,
+    struct Imu *const               imu,
```

Comment:
Hmm looks like you're missing a closing brace here

Reply:
nit; I'd place the Imu right below the Buzzer just to keep our hw objects together

Reply:
ok